src/job.rs

```diff
@@ -619,6 +621,37 @@ impl Job {
         }
     }
 
+    /// Read the receipients from old emails sent by the user user and add them as contacts.
```

Comment:
"recipients", "user user", "adds", looks like a copy of the message in the python test.